.travis.yml

```diff
@@ -19,6 +21,8 @@ before_script:
 - "./scripts/run_app.sh"
 script:
 - "./scripts/run_tests.sh"
+# fail travis if under X% test coverage
+# - coverage report --fail-under=80
```

Comment:
Might as well enable this, but set it to the around current code coverage level (maybe 75%)?